tests/test_mock_controller.py

```diff
@@ -303,7 +305,8 @@ async def test_stopEl(self):
 
     async def test_stop(self):
         target_azimuth = math.radians(10)
-        await self.write(f"moveAz:\n azimuth: {target_azimuth}\n")
+        target_rate = math.radians(0.1)
+        await self.write(f"moveAz:\n azimuth: {target_azimuth}\n azRate: {target_rate}")
```

Comment:
A central method to send commands would allow you to change the code in one place if the format changes again, and would make this code easier to follow -- i.e. just changing the existing `write` method to take a command and `**parameters`.

Reply:
I was working on that but then this ticket gained priority so I parked the other one. See

https://jira.lsstcorp.org/browse/DM-25130